index.html

```diff
@@ -1127,6 +1163,150 @@ <h2 property="schema:name">Discovering a complete Solid profile</h2>
+                            retrieve Solid WebID Profiles.</p>
+                    </div>
+                    <p about="" id="reader-application-extended-profile" rel="spec:requirement"
+                        resource="reader-application-extended-profile">
+                        <span property="spec:statement"><a rel="spec:requirementSubject" href="#ReaderApplication">Reader
+                                Application</a> <span rel="spec:requirementLevel" resource="spec:MAY">MAY</span> retrieve a
+                            representation of an <a href="#extended-profile-documents">Extended Profile Document</a>.</span>
+                    </p>
+                </section>
+                <section id="updating-profile" inlist="" rel="schema:hasPart" resource="#updating-profile">
+                    <h3 property="schema:name">Updating Profile</h3>
+                    <div datatype="rdf:HTML" property="schema:description">
+                        <p><a href="#writer-applications">Writer Applications</a> perform <a href="https://solidproject.org/TR/protocol#writing-resources">write operations</a> against Solid Protocol <a href="https://solidproject.org/TR/protocol#Server">servers</a>.</p>
+                        <div class="note" id="prioritize-webid-profile" inlist="" rel="schema:hasPart"
+                            resource="#prioritize-webid-profile">
+                            <h5 property="schema:name"><span>Note</span>:</h5>
+                            <div datatype="rdf:HTML" property="schema:description">
+                                <p>To promote self-describing resources and efficient discovery and reuse of profile
+                                    information, implementations and authors are encouraged to prioritize using the <a
+                                        href="#solid-profile">Solid WebID Profile</a> before needing to use the <a
+                                        href="extended-profile-documents">Extended WebID Profile.</a></p>
+                            </div>
+                        </div>
+                        </div>
+                        <p about="" id="writer-application-webid-profile" rel="spec:requirement"
+                            resource="writer-application-webid-profile">
+                            <span property="spec:statement"><a rel="spec:requirementSubject" href="#WriterApplication">Writer
+                                    Application</a> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> use an
+                                an <code>HTTP PUT</code> or <code>PATCH</code> request targeting the Solid WebID Profile to be
+                                updated.</span>
+                        </p>
+                        <p about="" id="protected-properties" property="schema:description">
+                            Solid WebID Profile might include protected properties, such as <code>solid:oidcIssuer</code>.
```

Comment:
Since statements with the `solid:oidcIssuer` predicate is defined as *protected* in this spec, I assume it only applies to resources that are WebID Profiles. How the solid storage/resource server is going to determine that a given resource should be handled as a WebID Profile?

Reply:
Created https://github.com/solid/webid-profile/issues/100 to follow up on this.